### ISSUE TITLE
Is there an example to create a Push subscription using Sharepoint REST API so that I get notified only when I am @mentioned in Sharepoint ?

### ISSUE THREAD

**Issue by nehaj910** [SharePoint/sp-dev-docs#2841]
#### Category
- [x] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

Is there an example to create a Push subscription using Sharepoint REST API so that I get notified only when I am @mentioned in Sharepoint ? I am creating an application that generates toast notification in Windows Action Center when I am @mentioned in Sharepoint.

**Comment by Seiyasu** [comment]
Would you please provide what you have tried.

**Comment by nehaj910** [comment]
I was able to find examples on how to create SharePoint Push Notifications for Windows apps but it didn't have any filtering by @mentions. Also I found some example on getting Sharepoint feed that contains posts of a user mentioned in it. But I wasn't able to find an example of  Sharepoint REST APIs to subscribe for @mentions Push Notifications. 
1. Create Push Notifications  in SharePoint apps for Windows Phone: https://docs.microsoft.com/en-us/sharepoint/dev/general-development/how-to-configure-and-use-push-notifications-in-sharepoint-apps-for-windows
2. Sharepoint feed that contains posts that mention the current user: https://docs.microsoft.com/en-us/sharepoint/dev/general-development/social-feed-rest-api-reference-for-sharepoint#bk_myMentionFeed
